app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -431,7 +436,8 @@ impl component::Model for Model {
         entries.set_background_color(HOVER_COLOR);
         entries.show_background_shadow(false);
         entries.set_background_corners_radius(0.0);
-        entries.hide_selection();
+        // FIXME(#182194574): Hide selection again once we have a proper selection box.
```

Comment:
1. Add a real link to the issue pls, it is easier to navigate by cliking in intellij idea then.
2. Every fixme should describe shortly WHY it is not working right now. The link should be an additional explanation. Right now, after reading this fixme, I dont understand why we cant enable it right now and what does "proper selection box" mean.

Reply:
I decided to revert this change, as with new styles default selection looks absolutely ugly.